src/device.rs

```diff
@@ -135,6 +143,21 @@ impl<SPI, NCS, E, EO> Device for SpiDevice<SPI, NCS>
         self.ncs.set_high().map_err(|a| SpiError::NCSError(a))?;
         Ok(())
     }
+
+    fn write_many(&mut self,
+                  reg: Register,
+                  buffer: &[u8])
+                  -> Result<(), Self::Error> {
+        if buffer.len() > 16 {
+            return Err(Self::Error::WriteManyError);
```

Comment:
Not sure about this: `write_many` is not public, so error here always means library author error, failed assertion. But panicking here is even worse, so I guess we should sticki with this, until compile-time solution is ready.